Configurations/windows-makefile.tmpl

```diff
@@ -381,6 +381,9 @@ distclean: clean
 	-del /Q /F makefile
 
 depend:
+	@ rem {- output_off() if $disabled{makedepend}; "" -}
+	@ $(PERL) $(SRCDIR)\util\add-depends.pl
```

Comment:
In other cases `$(PERL)` is quoted. One can also wonder if one should quote `$(SRCDIR)\...` It is quoted in some cases, but not others...

Reply:
Fixed in #5608